File: SDKs/sportsdata.io/golf-v2/1.0/go/pkg/models/operations/player.go
package operations

type PlayerFormatEnum string

const (
	PlayerFormatEnumXML  PlayerFormatEnum = "XML"
	PlayerFormatEnumJSON PlayerFormatEnum = "JSON"
)

type PlayerPathParams struct {
	Format   PlayerFormatEnum `pathParam:"style=simple,explode=false,name=format"`
	Playerid string           `pathParam:"style=simple,explode=false,name=playerid"`
}

type PlayerRequest struct {
	PathParams PlayerPathParams
}

type PlayerResponse struct {
	ContentType string
	Player      *interface{}
	StatusCode  int64
}

File: SDKs/sportsdata.io/csgo-v3-scores/1.0/python/src/sdk/models/operations/player.py
from dataclasses import dataclass, field
from typing import Any,Optional
from enum import Enum

class PlayerFormatEnum(str, Enum):
    XML = "xml"
    JSON = "json"


@dataclass
class PlayerPathParams:
    format: PlayerFormatEnum = field(metadata={'path_param': { 'field_name': 'format', 'style': 'simple', 'explode': False }})
    playerid: str = field(metadata={'path_param': { 'field_name': 'playerid', 'style': 'simple', 'explode': False }})
    

@dataclass
class PlayerRequest:
    path_params: PlayerPathParams = field()
    

@dataclass
class PlayerResponse:
    content_type: str = field()
    status_code: int = field()
    player: Optional[Any] = field(default=None)
    
